client/src/featureform/register.py

```diff
@@ -2684,15 +2684,15 @@ def get_training_set(self, name, variant=None, local=False):
         ```
 
         ``` json title="Output"
-        // get_training_set returns the TrainingSet object
+        // get_training_set returns the TrainingSet object from featureform.Resources
```

Comment:
Do tey need to know this?

Reply:
TrainingSet is ambiguous and could refer to a lot of different things

Reply:
Do we have multiple TrainingSet objects?